docs/modules/secret-operator/pages/secretclass.adoc

```diff
@@ -119,8 +119,61 @@ spec:
 
 *Format*: Free-form
 
-A Kubernetes `Secret` object is selected based on the xref:scope.adoc[scopes] specified on the `Volume`. Each field in this `Secret` is mapped
-to one file. It is suggested these `Secret` objects should follow one of the xref:#format[formats defined in this document].
+This backend can be used to mount `Secret` across namespaces into pods. The `Secret` object is selected based on two things:
+
+1. The xref:scope.adoc[scopes] specified on the `Volume` using the attribute `secrets.stackable.tech/scope`.
+2. The label `secrets.stackable.tech/class` specified in the `Secret` it's self. This must match the name of the `SecretClass`.
+
+Each field in this `Secret` is mapped to one file. It is suggested these `Secret` objects should follow one of the xref:#format[formats defined in this document].
+
+In the example below, given the three object definitions for a `Pod`, a `SecretClass` and a `Secret`, the operator will first read the Pod's volume attributes then look up the secret class. The `k8sSearch` backend will look up the secret object labeled with `secrets.stackable.tech/class: admin-credentials-class` (the name of the secret class) and mount the fields of the secret as files into the the container at the specified mount point (`/credentials`).
+
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Pod
+metadata:
+  name: my-app
+  labels:
+    app.kubernetes.io/name: my-app
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: my-app
+  containers:
+    #... skipped for brevity
+    volumeMounts:
+    - name: admin-credentials-volume
+      mountPath: /credentials
+  volumes:
+  - name: admin-credentials-volume
+    csi:
+      driver: secrets.stackable.tech
+      volumeAttributes:
+        secrets.stackable.tech/class: admin-credentials-class
+        secrets.stackable.tech/scope: pod
```

Comment:
CSI is deprecated (see https://github.com/stackabletech/secret-operator/pull/125)
```suggestion
  - name: admin-credentials-volume
    ephemeral:
      volumeClaimTemplate:
        metadata:
          annotations:
            secrets.stackable.tech/class: admin-credentials-class
            secrets.stackable.tech/scope: pod
        spec:
          storageClassName: secrets.stackable.tech
          accessModes:
            - ReadWriteOnce
          resources:
            requests:
              storage: "1"
```